cmd/maya-apiserver/app/command/start.go

```diff
@@ -183,6 +183,11 @@ func Run(cmd *cobra.Command, c *CmdStartOptions) error {
 		}
 	}()
 
+	// start webhook controller
+	go func() {
+		webhook.Start()
```

Comment:
`go webhook.Start()`?

Reply:
both are same , I like the other way to represent.. :smiley:  .
Anyways for time being, not going to start a goroutine  with-in maya-apiserver .

Reply:
both are same , I like the other way to represent.. :smiley:  .
Anyways for time being, not going to start a goroutine  with-in maya-apiserver .